docs/source/markdown/podman-push.1.md.in

```diff
@@ -28,7 +28,11 @@ $ podman push docker://quay.io/podman/stable
 
 # Push to a container registry with another tag
 $ podman push myimage quay.io/username/myimage
+```
+
+**podman push** can also be used as an alternative to **podman save**.  For example:
```

Comment:
I was looking for a wrap up note to the original "For example:".  I suggest this.

```suggestion
**podman push** can also be an alternative to **podman save**.  For instance, the following push commands save the image at the specified destination.
```